tests/mocks/fixtures/recommendations.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  createRecommendResponse,
+  createSingleSearchResponse,
+} from '../createAPIResponse';
+import { createSearchClient } from '../createSearchClient';
+
+type Options = {
+  /**
+   * Returns only the `objectID` attribute of the hits when `true`.
+   *
+   * @default false
+   */
+  minimal?: boolean;
+};
+
+const fixture = [
+  {
+    _highlightResult: {
+      name: {
+        matchLevel: 'none' as const,
+        matchedWords: [],
+        value: '<em>Moschino Love</em> – Shoulder bag',
+      },
+    },
+    name: 'Moschino Love – Shoulder bag',
+    objectID: '1',
+  },
+  {
+    _highlightResult: {
+      name: {
+        matchLevel: 'none' as const,
+        matchedWords: [],
+        value: '<em>Bag</em> “Sabrina“ medium Gabs',
+      },
+    },
+    name: 'Bag “Sabrina“ medium Gabs',
+    objectID: '2',
+  },
+];
+
+export function createMockedSearchClientWithRecommendations(
```

Comment:
```suggestion
export function createRecommmendSearchClient(
```

also I wonder, couldn't this simply be part of the existing `createSearchClient` with the option `minimalRecommendations`?

Reply:
I didn't want to change how `createSearchClient` doesn't provide any fixture for search or recommendations. I thought about only sharing the fixture, but wrapping those with `createMultiSearchResponse` and `createSingleSearchResponse` on each test would not be as beneficial as moving those to the shared entity as well.

Reply:
ok that makes sense, although I wouldn't be against having a fixture for search to avoid repetition too in the future